package.json

```diff
@@ -42,10 +42,12 @@
     "html-minifier": "^3.5.8",
     "imageinfo": "^1.0.4",
     "jsdom": "^11.11.0",
+    "lines-and-columns": "^1.1.6",
     "lodash": "^4.11.2",
     "memoizesync": "1.1.1",
     "mkdirp": "^0.5.1",
     "normalizeurl": "^1.0.0",
+    "opentype": "github:bramstein/opentype",
```

Comment:
Can we get on an officially released version? Do we need to poke Bram?

Reply:
What is it doing? You might be better of using something like fontkit.

Reply:
All I need is a tiny part of the cmap information. Namely which codepoints are present in the font file. I couldn't find that in the other libraries I tried. At least not for woff/woff2

Reply:
This package would be the exact thing I needed if it supported a wider range of font formats. https://www.npmjs.com/package/font-cmap

Reply:
ah, it does look like I can get that with `fontkit.openSync(tempWrite.sync(fontUsage.subsets.woff)).characterSet`. Their docs do require quite specific reading through. No good keywords to search on